docker-compose.yml

```diff
@@ -27,3 +27,15 @@ services:
           - "27017:27017"
       volumes:
           - /var/lib/mongodb
+
+    haproxy:
+      image: haproxy:1.7
+      ports:
+          - "80:80"
+      volumes:
+          - /c/projects/proxies/localhost/etc/haproxy:/usr/local/etc/haproxy/
```

Comment:
I don't. I'm the lightning rod that hits the failure every time people hard-code this path and it's happened quite a few times.